### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
Ran the site through a link checker. This PR does a few things:

* Updates a few external URLs that responded with 301 redirects
* Update internal links to point to new page locations
* Update images whose named weren’t slugified
* Adds a missing page (UCAS course examples)
* Fixes image resizing script so that it only resizes image formats Sharp can transform
* Fixes incorrect parameter in the `appImage` component (`src` wasn’t correct, `srcset` was)
* Adds a sitemap at `/sitemap` (a link to this is shown instead of a search box when no JS)

Then I went a bit crazy and also…
* Removed empty `tags:` fields
* Used consistent spacing for markdown lists
* Ensured (fairly) consistent syntax for screenshots
* Updated a few titles to be consistent

### CODE REVIEW

File: app/posts/publish-teacher-training-courses/2018-06-06-school-direct-view.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ School Direct routes are the most complex. List schools, show multiple UCAS cour
 We tested this with West London Teaching School Alliance:
 [research session](https://lookback.io/watch/mbc9BNqBJjoRkinAE). We also demoed it to groups of School Direct and SCITT providers in Brighton and Reading. Some provider user needs have been derived from these sessions: [provider user needs](https://docs.google.com/document/d/1Jb5uDZBnSFGcCdl3gAF9ggAo1klsEWL-KeSIGGLJpBE/edit)
 
-An iteration of this was tested with [Kingston School Direct](/publish-teacher-training-courses/kingston-school-direct)
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2018-06-06-school-direct-view.md]: This one still needs to be ported.

File: app/posts/publish-teacher-training-courses/2018-08-09-no-more-folding.md [hunk 2]
```diff
@@ -14,8 +14,8 @@ The folding exercise was ran while we were unclear what data, if any, we’d get
 
 We tested folded courses with training providers:
 
-*   [West London Teaching School Alliance](https://lookback.io/watch/mbc9BNqBJjoRkinAE) ([screenshots](/publish-teacher-training-courses/school-direct-view))
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2018-08-09-no-more-folding.md:17]: This link should still work?

File: app/posts/publish-teacher-training-courses/2018-08-14-formatting.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ The problem with these sorts of links is that they aren’t accessible. They don
 
 ## HTML
 
-We could allow users to use raw HTML: <a href="/blah">Link</a>. This is prone to more user error than markdown.
+We could allow users to use raw HTML: <a href="#">Link</a>. This is prone to more user error than markdown.
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2018-08-14-formatting.md]: Is this getting rendered as HTML? Probably should encode the `<>` etc instead.

**Review by fofr — APPROVED** [review]
Awesome work. :100:

Can you include details of the link checker you used, so we can run again easily in future?

File: app/posts/publish-teacher-training-courses/2018-06-06-school-direct-view.md (hunk 1 above)
Comment (paulrobertlloyd) [reply app/posts/publish-teacher-training-courses/2018-06-06-school-direct-view.md]: Link text restored. Will create an issue for porting this page over.

File: app/posts/publish-teacher-training-courses/2018-08-09-no-more-folding.md (hunk 2 above)
Comment (paulrobertlloyd) [reply app/posts/publish-teacher-training-courses/2018-08-09-no-more-folding.md:17]: Huh, it does. Restored!

File: app/posts/publish-teacher-training-courses/2018-08-14-formatting.md (hunk 3 above)
Comment (paulrobertlloyd) [reply app/posts/publish-teacher-training-courses/2018-08-14-formatting.md]: Using inline code to format this example now.

**Review by paulrobertlloyd — COMMENTED** [review]
Here’s the link checker I used: https://peacockmedia.software/mac/integrity/

There’s the also possibility of integrating [hyperlink](https://www.npmjs.com/package/hyperlink) into build checks, but that currently [doesn’t recognise ‘pretty’ URLs](https://github.com/Munter/hyperlink/pull/168).

**Pull request merged**